analyses/cms-open-data-ttbar/README.md

```diff
@@ -20,11 +20,41 @@ This directory is focused on running the CMS Open Data $t\bar{t}$ analysis throu
 | utils/config.py               | This is a general config file to handle different options for running the analysis.                               |
 | utils/hepdata.py              | Function to create tables for submission to the [HEP_DATA website](https://www.hepdata.net) (use `HEP_DATA = True`)     |
 
+#### Setting up the environment
```

Comment:
Later on in this README there are instructions to install `hepdata_lib` and `hepdata-cli`

https://github.com/iris-hep/analysis-grand-challenge/blob/main/analyses/cms-open-data-ttbar/README.md?plain=1#L59-L61

At the moment these are not in the `pixi` manifest and lock file, but they can be added if it would be useful.

Reply:
I think that's ok as-is!